### PR TITLE
Maddie/profile extension cleanup

### PR DESCRIPTION
**Pull request by maddiechase**
filters added for profile values

### CODE REVIEW

**Comment by maddiechase** [comment]
Need to add profiles to purchases

**Comment by maddiechase** [comment]
Need to add 
extensions/values to the new endpoints
jobs, staff, contracts, prospects, assets

File: source/includes/endpoints/_assets.md [hunk 1]
```diff
@@ -105,14 +109,14 @@ If any of the fields labelled optional above are empty, they will not be display
...
-| **object_link_fields** | hash | A hash of [asset object link fields](#asset-object-link-fields-beta). |
+| **object_link_fields** | hash | A hash of [asset object link fields](#asset-object-link-fields). |
 | standing | string | The standing of the asset type. |
 | has_manager | binary | Whether the asset type has a manger assigned to it. |
 | has_affiliation | binary | Whether the asset type has an affiliation assigned to it. |
 | has_address | binary | Whether the asset type has an address assigned to it. |
 | ordering  | unsigned | An integer representing the asset type's ordering on the deployment. |
 
-#### Asset Object Link Fields (Beta)
+#### Asset Object Link Fields )
```
Comment (matthewhughes112) [inline source/includes/endpoints/_assets.md]: Trailing `)` here

File: source/includes/endpoints/_assets.md [hunk 2]
```diff
@@ -588,8 +671,22 @@ This request returns a list of [extension fields](#the-extension-field-object) a
...
+
+
+
+### List all Extension Field Values on Assets
+> See the [extension section](#list-extension-values) for an example
+
+`GET /assets/extensions/values`
 
+This request returns a list of [extension field values](#the-extension-value-object) on [assets](#the-asset-object) This
+is the request [`GET /{object}/extensions/values`](#list-profile-values), where the object is "assets".
```
Comment (matthewhughes112) [inline source/includes/endpoints/_assets.md]: Typo: `list-profile-values` in link should point towards link extension values endpoint. This appears to be on other new endpoints too.

File: source/includes/endpoints/_purchases.md [hunk 3]
```diff
@@ -151,8 +179,47 @@ CURL -X get \
...
+
+
+
+### List Purchase Profile Fields
+> See the [profiles section](#retrieve-a-list-of-profile-fields) for a sample request
+
+`GET /purchases/profiles/fields`
+
+This request returns a list of [profile fields](#the-profile-field-object) available for any [purchase](#the-purchase-object)
+This is the request [`GET /{object}/profiles/values`](#list-profile-values), where the object is "purchases".
```
Comment (matthewhughes112) [inline source/includes/endpoints/_purchases.md]: Typo: wrong link here also.

File: source/includes/endpoints/_profiles.md [hunk 4]
```diff
@@ -202,6 +202,49 @@ This request supports requesting additional fields and linked objects from the [
 using the [`_fields`](#configuring-the-response-fields) parameter.
 
 
+### Basic Filter
```
Comment (matthewhughes112) [inline source/includes/endpoints/_profiles.md]: Have we documented filters for profile fields too? If not, do we want to add them as part of this review too?

File: source/includes/endpoints/_assets.md (hunk 1 above)
Comment (matthewhughes112) [reply source/includes/endpoints/_assets.md]: fixed fe8a2ac

File: source/includes/endpoints/_assets.md (hunk 2 above)
Comment (matthewhughes112) [reply source/includes/endpoints/_assets.md]: fixed f13e051

File: source/includes/endpoints/_purchases.md (hunk 3 above)
Comment (matthewhughes112) [reply source/includes/endpoints/_purchases.md]: also fixed f13e051

File: source/includes/endpoints/_profiles.md (hunk 4 above)
Comment (maddiechase) [reply source/includes/endpoints/_profiles.md]: We won't be adding filters on profile fields at this time

File: source/includes/endpoints/_contracts.md [hunk 5]
```diff
@@ -6,7 +6,7 @@ Contracts (also known as retainers) are objects for managing recurring client wo
 invoicing - such as license fee. A contract is created against a company/client record (that is, an
 [affiliation](#affiliations)), and work is tracked against the contract through [periods](#the-contract-period). You can
 configure your contract to automatically create periods at certain intervals, and invoice up-front for a fixed amount or
-at the end of the period for the work done. See the [supportdocumentation](#https://www.accelo.com/resources/help/guides/user/modules/retainers/) 
+at the end of the period for the work done. See the [support documentation](#https://www.accelo.com/resources/help/guides/user/modules/retainers/)
```
Comment (matthewhughes112) [inline source/includes/endpoints/_contracts.md]: Unrelated fix, but there should be no `#` in this link

File: source/includes/endpoints/_issues.md [hunk 6]
```diff
@@ -1207,6 +1207,17 @@ values), where the object is "issues", and whose id is the `{issue_id}`.
 
 
 
+### List all Extension Field Values on Issues
+> See the [extension section](#list-extension-values) for an example
+
+`GET /issues/extensions/values`
+
+This request returns a list of [extension field values](#the-extension-value-object) on [issues](the-issue-object).
```
Comment (matthewhughes112) [inline source/includes/endpoints/_issues.md]: Missing `#` in link here

**Pull request merged**